packages/linzjs-s3fs/src/file.local.ts

```diff
@@ -29,8 +39,8 @@ export class FsLocal implements FileProcessor {
         }
     }
 
-    async exists(filePath: string): Promise<boolean> {
-        return await new Promise((resolve) => fs.exists(filePath, resolve));
+    exists(filePath: string): Promise<boolean> {
```

Comment:
Is this suppose to be a async function? and `await` before `head()`?

Reply:
since we no longer "await" anything in the function its doest have to be async